src/mixins/enum-mixin.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Vue } from 'vue-property-decorator'
+import { EntityTypes } from '@/enums'
+
+/**
+ * Mixin that provides some useful enum-related utilities.
+ */
+@Component({})
+export default class EnumMixin extends Vue {
+  /**
+   * Converts the entity type to a numbered corp description.
+   * @param type the entity type to convert
+   * @returns the numbered corp description
+   */
+  entityTypeToNumberedDescription (type: EntityTypes): string {
+    switch (type) {
+      case EntityTypes.BC_COMPANY: return 'Numbered Company'
+      case EntityTypes.BC_CORPORATION: return 'Numbered Corporation'
+      case EntityTypes.BC_ULC_COMPANY: return 'Numbered Unlimited Liability Company'
+      case EntityTypes.BENEFIT_COMPANY: return 'Numbered Benefit Company'
+      case EntityTypes.COOP: return 'Numbered Cooperative'
+    }
+    return 'Unknown' // should never happen
```

Comment:
Would a `default` case do the same thing?

Reply:
Yes it would.
Again, this is probably my personal code style, but for methods that return a value I like seeing a return statement at the end of it. This can avoid mistakes made by placing extra code after a switch statement and expecting it to run in the default case.

Reply:
Sounds good!